### DISCUSSION TITLE
Styling home page like the Hextra landing page

### DISCUSSION THREAD

**Discussion by huntsyea** [imfing/hextra#369] [Q&A] [answered]
I would like to use a version of the hextra landing page [https://imfing.github.io/hextra/](https://imfing.github.io/hextra/) for my home page. I have tried mirror the hextra-home.html in layouts but the styling keeps breaking it.

**Comment by imfing** [comment, marked as answer]
@huntsyea which version are you using? for example, with v0.7.3, you can follow the example:

https://github.com/imfing/hextra/blob/v0.7.3/exampleSite/content/_index.md?plain=1

**Reply by huntsyea** [reply]
I figured it out! This worked, thank you
